.changeset/empty-adults-allow.md

```diff
@@ -0,0 +1,53 @@
+---
+"@salt-ds/core": minor
+---
+
+- Promote `Dialog` to Core
+- `Dialog` is a window that opens over the application content, focusing the user’s attention on a particular task or piece of information.
+- It can communicate new information, errors, warnings, or successful completion of a process or task
+
+```tsx
+const AlertDialog = () => {
+  const [open, setOpen] = useState(openProp);
+
+  const handleRequestOpen = () => {
+    setOpen(true);
+  };
+
+  const onOpenChange = (value: boolean) => {
+    setOpen(value);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <Button data-testid="dialog-button" onClick={handleRequestOpen}>
```

Comment:
Try remove `data-testid` for buttons? Shouldn't be needed in all tests and examples